examples/synthetics_mesh_example_test.go

```diff
@@ -288,24 +288,29 @@ func toPercent(v float64) float64 {
 	return v * 100
 }
 
-func formatMetricData(md *syntheticspb.MetricData) string {
+func formatMetricData(md *syntheticspb.MetricData, isCurrentMeasurementValid bool) string {
 	if md == nil {
 		return "[X]\t"
 	}
 
 	return fmt.Sprintf(
 		"%v / %v / %v / %v\t",
-		formatMetricValue(md.GetCurrent()),
-		formatMetricValue(md.GetRollingAvg()),
-		formatMetricValue(md.GetRollingStddev()),
+		formatMetricValue(md.GetCurrent(), isCurrentMeasurementValid),
+		formatMetricValue(md.GetRollingAvg(), isCurrentMeasurementValid),
+		formatMetricValue(md.GetRollingStddev(), isCurrentMeasurementValid),
 		md.GetHealth(),
 	)
 }
 
 // formatMetricValue formats the value of metric given in nanoseconds to milliseconds.
-func formatMetricValue(metricValue uint32) string {
-	if metricValue == 0 {
+func formatMetricValue(metricValue uint32, isCurrentMeasurementValid bool) string {
+	if metricValue == 0 && !isCurrentMeasurementValid {
```

Comment:
Well, `stddev` and `rolling_avg` may be valid even if the current measurement in not (due to 100% packet loss). Especially for jitter, 0 is really a possible albeit unlikely value. It is an impossible value for latency (and of course a very valid value for packet loss).

Let's do this:
- print stddev and rolling_avg values always (regardless of value and/or validity of the current measurement)
- print current latency and jitter measurements only if packet loss is < 100%, regardless of measured value

We do not need to spend whole lot more time on this, since this is after all just an example.